contracts/Registrar.sol

```diff
@@ -134,13 +189,28 @@ contract Registrar is OpsManaged {
 			_stakingIntentHash, _unlockSecret);
 	}
 
-	/*
-	 *  OpenSTUtility
+	/**
+	 *  @notice External function confirmStakingIntent.
+	 *
+	 *  @dev Callable only by Ops.
+	 *
+	 *  @param _uuid Uuid of the token.
+	 *  @param _staker Address of the staker.
+	 *  @param _stakerNonce Nonce of the staker.
+	 *  @param _beneficiary Beneficiary address on utility chain.
+	 *  @param _amountST Amount OST staked.
+	 *  @param _amountUT Amount utility tokens to be minted.
+	 *  @param _stakingUnlockHeight Height upto which stake is locked.
+	 *  @param _hashLock Hash lock for the staking intent.
+	 *  @param _blockHeight Current block height.
```

Comment:
I don't believe "current" is correct (because this block height is the value included in the staking intent hash which comes from the value chain).